tests/ga/test_extension.py

```diff
@@ -3425,6 +3425,31 @@ def mock_http_put(url, *args, **_):
 
             self.assertEqual(expected_status, actual_status_json)
 
+    def test_ext_handler_reports_proper_status_when_dependency_fails(self):
```

Comment:
This test succeeds even if I comment out the changes you did above. I'm not sure what exactly you're trying to test here.